syntax = "proto3";

package recommender;

service Recommender{
	rpc Recommend(WhichUser) returns (UserList){}
}

message WhichUser{
  string userID = 1;
}
message UserList{
  string userID = 1;
  repeated string recommendedID = 2;
}
